api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1297,6 +1404,20 @@ type ClusterAutoscaling struct {
 	//
 	// +optional
 	PodPriorityThreshold *int32 `json:"podPriorityThreshold,omitempty"`
+
+	// expanders guide the autoscaler in choosing node groups during scale-out.
+	// Sets the order of expanders for scaling out node groups.
+	// Options include:
+	// * LeastWaste - selects the group with minimal idle CPU and memory after scaling.
+	// * Priority - selects the group with the highest user-defined priority.
+	// * Random - selects a group randomly.
+	// If not specified, `LeastWaste` is the default.
```

Comment:
Nit, the default should be represented a list, not a single string IMO
```suggestion
	// If not specified, `[LeastWaste]` is the default.
```